lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -51,6 +51,11 @@ check_page_contains:
   element:
     selector:
       xpath: //*[contains(.,'<content>')]
+check_page_not_contains:
+  element:
+    selector:
+      xpath: //*[contains(.,'<content>')]
+    missing: true
```

Comment:
If we want to add action `check_page_not_match/contains`, we'd better place an anchor which looks like 
```
check_page_contains/match:
  element: &anchor
    xpath: ...
check_page_not_contains/match:
  element:
    <<: *anchor
  missing: true
```

Reply:
Good idea! Updated.